package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@notifee/react-native",
-  "version": "1.10.1",
+  "version": "1.10.2-0",
```

Not sure the `-0` is intentional? Conventionally if there were a sub-patch semver component it would have human-meaning like using alpha beta or rc and thus be `1.10.2-rc.1` or similar for pre-testing before a real release

```suggestion
  "version": "1.10.2",
```

---

yea, intentional,  this was just a prepatch not the actual release. I'm going to rebuild the binaries now because they were built with code 12.5

---

ace!